src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractPrepareTask.groovy

```diff
@@ -87,10 +88,14 @@ public class AbstractPrepareTask extends AbstractServerTask {
         }
 		
 		@Override
-		public void provideJsonFileDependency(File file) {
-			project.getConfigurations().create('jsonProvided')
-			DependencyHandler dependencies = project.getDependencies();
-			dependencies.add("jsonProvided", project.files(file))
+		public void provideJsonFileDependency(File file, String groupId, String version) {
```

Comment:
Update copyright year at top of file - change 2021 to 2022. Check other files for same update.